bindings/pyroot_experimental/cppyy/CMakeLists.txt

```diff
@@ -4,7 +4,11 @@
 # For the licensing terms see $ROOTSYS/LICENSE.
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
-set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")
+if(MSVC)
```

Comment:
This will be replace by the use of `${ROOT_LIBRARY_PROPERTIES}` by: https://github.com/root-project/root/pull/5643/commits/04354d7cebfcbb31d540936ac397b371effebf7c